src/components/lib/ConnectWalletDialog.js

```diff
@@ -22,12 +23,33 @@ const injectedConnector = new InjectedConnector({
 	supportedChainIds: SUPPORTED_CHAIN_IDS
 })
 
+const ledgerConnector = new LedgerConnector({
+	// We use the chainId of mainnet here to workaround an issue with the ledgerjs library.
+	// It currently throws an error for the default chainId of 1377 used by Geth/Ganache.
+	// 
+	// The `v` value in ECDSA sigs is typically used as a recovery ID, but we also encode it 
+	// differently depending on the chain to prevent transaction replay (the so called chainId of EIP155).
+	// 
+	// At some point, Ledger had to update their firmware, to swap from a uint8 chainId to a uint32 chainId [1].
+	// 
+	// They updated their client library with a 'workaround' [2], but it doesn't appear to work.
```

Comment:
This workaround actually seems to do the opposite, masking any >8-bit chain id to only keep the lowest 8 bits. Guessing it's designed not to screw up when communicating with the ledger, rather than being designed to support larger chain ids?

Reply:
> Guessing it's designed not to screw up when communicating with the ledger, rather than being designed to support larger chain ids?

Yup. Poor wording on my end, deriving from a poor understanding. Basically, Ledger has implemented 32 bit chainID support in their device firmware, but the Ledger Ethereum app still transports the truncated 8 bit ID, I'm guessing. eg. one chain, [Piri](https://github.com/LedgerHQ/ledger-app-eth/pull/18), has its 32 bit chainID working. 

That and my assumptions were wrong - the LedgerSubprovider from [0x uses an older version](https://sourcegraph.com/github.com/0xProject/0x-monorepo@2086bfff25b51be50c200e6523a8e876f5361fb6/-/blob/packages/subproviders/package.json#L42) of [@ledgerhq/hw-app-eth](https://sourcegraph.com/github.com/0xProject/0x-monorepo@2086bfff25b51be50c200e6523a8e876f5361fb6/-/blob/packages/subproviders/package.json#L42), without this workaround. Soooo, going to address that in a bit.